sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -573,8 +574,64 @@ def test_queue_by_queue_client_conn_str_receive_handler_receiveanddelete(self, u
                 for message in receiver:
                     messages.append(message)
                 assert len(messages) == 0
-    
-    
+
+            if not uamqp_transport:
```

Comment:
will it be better to make this a separate pyamqp only test ( skip on uamqp ) or keep it together with the old test?

Reply:
will move this to a separate test + skip on uamqp

Reply:
updated!